sharpy/rom/utils/librom.py

```diff
@@ -123,24 +127,73 @@ def balreal_direct_py(A, B, C, DLTI=True, Schur=False, full_outputs=False):
 
     ### Find transformation matrices
     # avoid Cholevski - unstable
-    hsv_sq, Tinv = np.linalg.eig(np.dot(Wc, Wo))
-    T = np.linalg.inv(Tinv)
 
-    # sort
-    iisort = np.argsort(hsv_sq)[::-1]
-    hsv = np.sqrt(hsv_sq[iisort])
-    T = T[:, iisort]
-    Tinv = Tinv[iisort, :]
+    # Added 11/04/2020: Building T and Tinv using SVD:
```

Comment:
```suggestion
    # Building T and Tinv using SVD:
```

It isn't really necessary to add the date in the code when you add changes since it is all tracked by git.